Quick_Deploy/HuggingFaceTransformers/README.md

```diff
@@ -0,0 +1,180 @@
+<!--
+# Copyright 2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions
+# are met:
+#  * Redistributions of source code must retain the above copyright
+#    notice, this list of conditions and the following disclaimer.
+#  * Redistributions in binary form must reproduce the above copyright
+#    notice, this list of conditions and the following disclaimer in the
+#    documentation and/or other materials provided with the distribution.
+#  * Neither the name of NVIDIA CORPORATION nor the names of its
+#    contributors may be used to endorse or promote products derived
+#    from this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS ``AS IS'' AND ANY
+# EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+# PURPOSE ARE DISCLAIMED.  IN NO EVENT SHALL THE COPYRIGHT OWNER OR
+# CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+# EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+# PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+# PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY
+# OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+# (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+-->
+
+# Deploying Hugging Face Transformer Models in Triton
+
+The following tutorial demonstrates how to deploy an arbitrary hugging face transformer
+model on the Triton Inference Server using Triton's [Python backend](https://github.com/triton-inference-server/python_backend). For the purposes of this example, three transformer
+models will be deployed:
+- [tiiuae/falcon-7b](https://huggingface.co/tiiuae/falcon-7b)
+- [adept/persimmon-8b-base](https://huggingface.co/adept/persimmon-8b-base)
+- [mistralai/Mistral-7B-v0.1](https://huggingface.co/mistralai/Mistral-7B-v0.1)
+
+These models were selected because of their popularity and consistent response quality.
+However, this tutorial is also generalizable for any transformer model provided 
+sufficient infrastructure. 
+
+*NOTE*: The tutorial is intended to be a reference example only. It is a work in progress.
```

Comment:
```suggestion
*NOTE*: The tutorial is intended to be a reference example only. It may not be tuned for optimal performance.
```

Reply:
Calling it a WIP may deter users from trying it, thinking it may not work.

Reply:
I agree. In that same vein then do we want to change the the WIP phrase for the vLLM [tutorial](https://github.com/triton-inference-server/tutorials/blob/512ba538fd81b37ad0cc3133a6d7905f72a9ad04/Quick_Deploy/vLLM/README.md?plain=1#L37C66-L37C91) to something like: 
"The tutorial is intended to be a reference example only and has known limitations."?

Reply:
sounds good to me CC @tanmayv25

Reply:
I went ahead and changed it. I can revert it if need be.